### PR TITLE
TRUS-4041 Update NonTaxableRegistration Page

### PR DESCRIPTION
**Pull request by carrielies**

### CODE REVIEW

File: app/views/register/suitability/NonTaxableTrustRegistrationAccessCodeView.scala.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
         @components.error_summary(form.errors)
 
-        @components.heading("nonTaxableTrustRegistrationAccessCode.heading")
+        <p>@components.heading_with_caption("nonTaxableTrustRegistrationAccessCode")</p>
```
Comment (jackheslop96) [inline app/views/register/suitability/NonTaxableTrustRegistrationAccessCodeView.scala.html:38]: Don't believe this needs to be inside a <p> element

File: conf/messages.en [hunk 2]
```diff
@@ -362,17 +362,26 @@ suitability.noNeedToRegister.p4 = You can
 suitability.noNeedToRegister.p4.link = return to register and maintain a trust for a client
 
 nonTaxableTrustRegistrationAccessCode.title = Private Beta
-nonTaxableTrustRegistrationAccessCode.heading = Private Beta
-nonTaxableTrustRegistrationAccessCode.p1 = In order to control the number of Non-Taxable Trust registrations, we have added limited access for the private beta. You will need to enter your provided access code and select ''Continue'' to enter the registration journey.
+nonTaxableTrustRegistrationAccessCode.caption = Private Beta
+nonTaxableTrustRegistrationAccessCode.heading = Register a non-taxable trust
+nonTaxableTrustRegistrationAccessCode.p1 = You can now use the Trust Registration Service to register non-taxable trusts. To register a non-taxable trust, enter the access code and select 'continue'
```
Comment (jackheslop96) [inline conf/messages.en:367]: The `'` style of apostrophe needs to be escaped with another one. So it should be `''continue''`

Comment (carrielies) [reply conf/messages.en]: Updated

File: app/views/register/suitability/NonTaxableTrustRegistrationAccessCodeView.scala.html (hunk 1 above)
Comment (carrielies) [reply app/views/register/suitability/NonTaxableTrustRegistrationAccessCodeView.scala.html]: Updated

**Pull request merged**